README.md

```diff
@@ -1,75 +1,61 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Module 1 Capstone
 
 > One paragraph statement about the project.
```

Comment:
- [ ] Please kindly give  a description of what this project is about, this will enable visitors and potential recruiters to quickly get an Idea of what the project is about.